libs/api/domains/cms/src/lib/models/linkGroup.model.ts

```diff
@@ -0,0 +1,23 @@
+import { Field, ObjectType } from '@nestjs/graphql'
+
+import { ILinkGroup } from '../generated/contentfulTypes'
+
+import { Link, mapLink } from './link.model'
+
+@ObjectType()
+export class LinkGroup {
+  @Field()
+  name: string
+
+  @Field(() => Link)
+  primaryLink: Link
+
+  @Field(() => [Link])
+  childrenLinks?: Array<Link>
+}
+
+export const mapLinkGroup = ({ fields }: ILinkGroup): LinkGroup => ({
+  name: fields.name ?? '',
+  primaryLink: mapLink(fields.primaryLink),
```

Comment:
`mapLink` will throw an error on undefined so we should check if `fields.primaryLink` exists before mapping